code/modules/mod/modules/modules_antag.dm

```diff
@@ -451,3 +451,60 @@
 	if(!holding_storage || holding_storage.max_specific_storage >= mod.w_class)
 		return
 	mod.forceMove(drop_location())
+
+/obj/item/mod/module/demoralizer
+	name = "MOD psi-echo demoralizer module"
+	desc = "One incredibly morbid member of the RND team at Roseus Galactic posed a question to her colleagues. \
+	'I desire the power to scar my enemies mentally as I murder them. Who will stop me implementing this in our next project?' \
+	And thus the Psi-Echo Demoralizer Device was reluctantly invented. The future of psychological warfare, today!"
+	icon_state = "brain_hurties"
+	complexity = 0
+	idle_power_cost = DEFAULT_CHARGE_DRAIN * 0.1
+	removable = FALSE
+	var/datum/proximity_monitor/advanced/demoraliser/demoralizer
+
+/obj/item/mod/module/demoralizer/on_suit_activation()
+	var/datum/demoralise_moods/module/mood_category = new()
+	demoralizer = new(mod.wearer, 7, TRUE, mood_category)
+
+/obj/item/mod/module/demoralizer/on_suit_deactivation(deleting = FALSE)
+	QDEL_NULL(demoralizer)
+
+/obj/item/mod/module/infiltrator
+	name = "MOD infiltration core programs module"
+	desc = "The primary stealth systems operating within the suit. Utilizing electromagnetic signals, \
+		the wearer simply cannot be observed closely, or heard clearly by those around them."
+	icon_state = "infiltrator"
+	complexity = 0
+	removable = FALSE
+	idle_power_cost = DEFAULT_CHARGE_DRAIN * 0
+	incompatible_modules = list(/obj/item/mod/module/infiltrator)
+	// The suits original wearable slot before installing the mod
+	var/old_slot
+	// The new wearable slot after installing the mod
+	var/new_slot = ITEM_SLOT_BELT
```

Comment:
why doesnt the mod just fit on the belt straight away?

Reply:
actually this is a module not a control, damn modsuit's names

i guess the better question would then be, why do we do this instead of make the infiltrator modsuit alone fit in the belt?

Reply:
Because I don't want to deal with what hardcoding control unit wearable slots will require, and aadditionally this is a better question for Fikou as I'm mimicking how plate compression handles weight.

Reply:
Although maybe I could handle it in the theme

Reply:
OH I'm a moron, there is a car already for this sorry willard.